ui/components/ui/tabs/flask/dropdown-tab/dropdown-tab.test.js

```diff
@@ -42,7 +42,7 @@ describe('DropdownTab', () => {
 
     fireEvent.click(element);
 
-    expect(onClick).toHaveBeenCalled();
-    expect(onChange).toHaveBeenCalled();
+    expect(onClick).toHaveBeenCalledWith(args.tabIndex);
```

Comment:
Does `onClick` trigger for both the initially selected element and then the newly selected element? If so, can we test it?